siac/cmds.go

```diff
@@ -12,25 +12,26 @@ import (
 var hostname = "http://localhost:9980"
 
 // helper function for reading http GET responses
-func getResponse(handler string, vals *url.Values) string {
+func getResponse(handler string, vals *url.Values) (map[string]interface{}, error) {
 	// create query string, if supplied
-	qs := "?"
```

Comment:
can this be renamed to `query`? `qs` is pretty opaque, even if the comment says query string. Just trying to minimize the amount of thinking required to read the code.